cgmanifest.json

```diff
@@ -189,6 +198,105 @@
             }
          }
       },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"103abd9df2e7ad57f45531349471ea5304a7efe5",
+               "repositoryUrl":"https://github.com/dotnet/corefxlab.git"
+            }
+         }
+      },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"18a36291e48808fa7ef2d00a764ceb1ec95645a5",
+               "repositoryUrl":"https://github.com/dotnet/standard.git"
+            }
+         }
+      },
+         {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"217f52fb121ef92491e5d5f71394b07ce4ead1d0",
+               "repositoryUrl":"https://github.com/KjellKod/g3log.git"
+            }
+         }
+      },
+        {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"50893291621658f355bc5b4d450a8d06a563053d",
+               "repositoryUrl":"https://github.com/madler/zlib.git"
+            }
+         }
+      },
+           {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"d264a2603493fecda607c1d1cda87fedba77d36b",
+               "repositoryUrl":"https://github.com/Microsoft/CNTK.git"
+            }
+         }
+      },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"9ff0cc0f0212cb1e85b83ce5d2907ba9531b6cae",
+               "repositoryUrl":"https://github.com/nlohmann/json.git"
+            }
+         }
+      },
+            {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"971e2e89d08deeae0139d3011d15646fdac13c92",
+               "repositoryUrl":"https://github.com/numpy/numpy.git"
+            }
+         }
+      },
+            {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"c3cceac115c072fb63df1836ff46d8c60d9eb304",
+               "repositoryUrl":"https://github.com/NVlabs/cub.git"
+            }
+         }
+      },
+            {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"90537289a04ef5d572496240e2ac3a881be518d2",
```

Comment:
Do we depend on pytorch?

Reply:
I believe this references the usage of caffe2-based operator implementations in a few cases.